### PR TITLE
Remove SWIG generated bindings 

### PR DESCRIPTION
**Pull request by matszczygiel**
This PR removed things related to SWIG binding generation and added the doc comments to the UDL file.
I haven't documented the events in UDL, since I feel the structure of events should be flattened (the `FinishEvent` should be inlined in the main event structure), and I'll address this in the subsequent PR

### CODE REVIEW

File: norddrop/src/norddrop.udl [hunk 1]
```diff
@@ -91,128 +174,325 @@ interface Event {
...
 
+	/// # Finishes the transfer from either side
+	///
+	/// # Arguments
+	/// * `transfer_id`: Transfer UUID
 	[Throws=Error]
 	void finish_transfer([ByRef] string transfer_id);
 
+	/// Removes a single transfer file from the database. The file must be rejected
+	/// beforehand, otherwise the error is returned.
```
Comment (LukasPukenis) [inline norddrop/src/norddrop.udl:450]: The file must be in terminal state, not only rejected: canceled, rejected, failed

File: norddrop/src/norddrop.udl [hunk 2]
```diff
@@ -91,128 +174,325 @@ interface Event {
...
+	/// # Arguments
+	/// * `peer` - Peer address.
+	/// * `descriptors` - transfer file descriptors.
+	///
+	/// # Returns
+	/// A String containing the transfer UUID.
 	[Throws=Error]
 	string new_transfer([ByRef] string peer, [ByRef] sequence<TransferDescriptor> descriptors);
 
+	/// # Finishes the transfer from either side
```
Comment (LukasPukenis) [inline norddrop/src/norddrop.udl:442]: I sort of just noticed but I think `finalize` would be even better as there's no way to misinterpret it, what do you think?

**Review by LukasPukenis — DISMISSED** [review]

File: norddrop/src/norddrop.udl (hunk 2 above)
Comment (matszczygiel) [reply norddrop/src/norddrop.udl:442]: Yeah, it makes sense :+1:

Comment (matszczygiel) [reply norddrop/src/norddrop.udl]: Done

File: norddrop/src/norddrop.udl (hunk 1 above)
Comment (matszczygiel) [reply norddrop/src/norddrop.udl]: Done

**Pull request merged**